### ISSUE TITLE
#list doesn't behave as it should?

### ISSUE THREAD

**Issue by nofxx** [ahoward/macaddr#26]
Forgot to thank you for this gem ;)

Mac.addr returns a string. "mac", and the singlemethod #list is not applied to it.
So I need:  Mac.addr.respond_to?(:list) . Can't get any uglier.

Also, a better approach here: https://github.com/tsilen/macaddr
The singleton method #list really doesn't make sense.

**Comment by jrmhaig** [comment]
Yes, please fix the list of all Mac addresses. Mac.addr.list does not work for Ruby 2.1.0 and later.
